examples/pinecone-vector-store/load-docs.ts

```diff
@@ -0,0 +1,66 @@
+// load-docs.ts
+import fs from "fs/promises";
+import {
+  SimpleDirectoryReader,
+  storageContextFromDefaults,
+  VectorStoreIndex,
+} from "llamaindex";
+import { PineconeVectorStore } from "../../../packages/core/src/storage/vectorStore/PineconeVectorStore";
```

Comment:
please also import this from "llamaindex" (works if you use the local build containing your class - see README.md)

Reply:
Done!